.travis.yml

```diff
@@ -25,6 +25,8 @@ script:
 - make
 - git diff --exit-code
 - git diff --cached --exit-code
+- mkdir -p tmp
```

Comment:
No. That makes sense. I thought it might have been an accidentally brought
along commit from `git checkout meat-cool-branch`. I do that all the time :)

💙👍🏼




-- 
Brandon Burton

On June 30, 2016 at 18:40:35, Dan Buch (notifications@github.com) wrote:

> In .travis.yml
> <https://github.com/travis-ci/packer-templates/pull/162#discussion_r69237098>
> :
>
> > @@ -25,6 +25,8 @@ script:
> >  - make
> >  - git diff --exit-code
> >  - git diff --cached --exit-code
> > +- mkdir -p tmp
>
> Kinda. Some work on another branch that depends on repo settings env vars
> makes this (or equiv) necessary in order to pass. Either that or we unset
> the repo settings env vars in the meantime (?)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/travis-ci/packer-templates/pull/162/files/59dc2ee64e78d9093bfc184e0d8b4e7e71d384a1#r69237098>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAlvPd9PBIdY_lC9WujDcbuk8GN0w6Wks5qRHATgaJpZM4JCpRG>
> .
>